src/main/pages/che-6/user-guide/ide-projects.adoc

```diff
@@ -10,25 +10,26 @@ folder: che-6/user-guide
 
 
 [id="default-location-of-projects"]
-== Default Location of Projects
+== Default location of projects
 
-When a workspace machine with a workspace agent starts, the IDE makes calls to Project API to get workspace projects. By default, they are located in `projects` in a machine container/pod. This location isn’t configurable as many other IDE components depend on it.
+When a workspace machine with a workspace agent starts, the IDE makes calls to the project API to get workspace projects. By default, projects are located in the `/projects` directory in a machine container or pod. You can not configure this location because other IDE components depend on it.
```

Comment:
The original capitalized "Project API" -- I assume that was intentional, and that the API is literally named "Project".

It's worth at least checking to make sure that the change being introduced here is accurate. :)